spec/presenters/instance_presenter_spec.rb

```diff
@@ -32,6 +26,40 @@
 
     expect(instance_presenter.site_contact_email).to eq "admin@example.com"
   end
+  
+  context "when Setting.open_registrations is false" do
+    it "closes registrations" do
+      Setting.open_registrations = false
```

Comment:
As far as I can tell none of the other specs adhere to this; is this something that should be being done across the board?